presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -42,6 +42,24 @@
     private double memoryRevokingThreshold;
     private double memoryRevokingTarget;
     private boolean retryOnOutOfMemoryBroadcastJoinEnabled;
+    private double multiplierForResourceAllocation = 1.25;
+    private PhysicalResourceAllocationStrategy physicalResourceAllocationStrategy = PhysicalResourceAllocationStrategy.DISABLED;
+    private QueryTypeByDuration queryTypeByDuration = QueryTypeByDuration.DISABLED;
+    private DataSize maxResourceUsagePerExecutor = new DataSize(10, GIGABYTE);
+
+    public enum PhysicalResourceAllocationStrategy
+    {
+        DISABLED,
+        OPTIMIZED_FOR_CPU,
+        OPTIMIZED_FOR_LATENCY
+    }
+
+    public enum QueryTypeByDuration
+    {
+        DISABLED,
+        SMALL,
+        LONG
+    }
```

Comment:
Let me think about if we can remove this config by having it to be `int TargetedRuntimeInMinutes`